### PR TITLE
Enable OOM tracking in periodic master perf test jobs

### PR DESCRIPTION
**Pull request by tosi3k**
Roll out the mechanism implemented in kubernetes/perf-tests#1309 to the rest of periodic master performance tests.

/sig scalability
/cc jkaniuk
/cc mm4tt
/cc wojtek-t

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml [hunk 1]
```diff
@@ -117,6 +117,7 @@ periodics:
       - --kubemark
       - --kubemark-nodes=100
       - --provider=gce
+      - --env=CL2_ENABLE_CLUSTER_OOMS_TRACKER=true
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml:120]: What does OOM tracker do in kubemark? Does it really do something useful?

IIUC, this is watching events from kubemark cluster, where NPD is faked, so it's baiscally no-op.

Comment (jkaniuk) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml:120]: It still tracks components on the master.

Comment (wojtek-t) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml:120]: ok - good point.
That said, we should probably make that explicit somewhere (measurement doc?) [that it only works for master components and we're fine with it.]

**Pull request merged**